core/src/main/java/tc/oc/pgm/projectile/ProjectileModule.java

```diff
@@ -69,6 +69,7 @@ public ProjectileModule parse(MapFactory factory, Logger logger, Document doc)
         Duration coolDown = XMLUtils.parseDuration(projectileElement.getAttribute("cooldown"));
         boolean throwable =
             XMLUtils.parseBoolean(projectileElement.getAttribute("throwable"), true);
+        boolean precise = XMLUtils.parseBoolean(projectileElement.getAttribute("precise"), false);
```

Comment:
Let's make `precise` default to `true`. Map makers can disable it.

Reply:
I think this is appropriate; there are little to no maps that use fireballs and "depend" on them being as janky as they tend to be.